src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -98,6 +138,28 @@ export const ActionPanelContent = ({
           />
         </ErrorBoundary>
       )}
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        <Button
+          sx={{ width: '95%', mt: '1rem' }}
+          variant="contained"
```

Comment:
ser ut som knappen er outlined i skissene? blir veldig dominant som contained
```suggestion
          variant="outlined"
```